doc/source/examples/local_launcher/sg_execution_times.rst

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
This is an auto-generated file, I think it shouldn't be committed (also applies to some of the other files here).